apps/condo/domains/common/components/GraphQlSearchInput.tsx

```diff
@@ -20,10 +20,12 @@ interface ISearchInputProps extends SelectProps {
     autoFocus?: boolean
     initialValue?: string
     formatLabel?: (option: { value: string, text: string }) => JSX.Element
+    researchDeps?: Array<any>
+    searchQueryArguments?: (selected, value) => string
 }
 
 export const GraphQlSearchInput: React.FC<ISearchInputProps> = (props) => {
-    const { search, onSelect, formatLabel, ...restProps } = props
+    const { search, onSelect, formatLabel, researchDeps, searchQueryArguments, ...restProps } = props
```

Comment:
What is researchDeps? Provide description or use more readable and understandable name.

Reply:
dependencies where re-search occurs

Reply:
rename to reSearchDependencies